web_crawler.py

```diff
@@ -0,0 +1,106 @@
+"""search for urls and save their html content to files asynchronously"""
+import asyncio
+import aiohttp
+import constants
+from bs4 import BeautifulSoup
+
+
+async def fetch(session, url):
```

Comment:
Separate the functions related to the crawling to another module. Rename the main module to `main.py`